frontend/src/components/common/with-header/components/header/components/nav-notification/nav-notification.tsx

```diff
@@ -150,6 +141,65 @@ export const NavNotification: React.FC = () => {
                 {isExpanded ? 'Hide' : 'Show'} all messages
               </span>
             </div>
+
+            <div
+              className="d-flex justify-content-center align-items-center p-1"
+              onClick={onReadAll}
+            >
+              <span className={getAllowedClasses(styles.footerText)}>
+                Make all read
```

Comment:
```suggestion
                Mark all as read
```
perhaps something like this will be better?

Reply:
this is part of the old tested code, perhaps I'll leave it as it was